lib/ast-plugins.js

```diff
@@ -28,19 +29,30 @@ module.exports = {
     var Compiler = require(templateCompilerPath);
     var templateCompilerFullPath = require.resolve(templateCompilerPath);
     var templateCompilerCacheKey = fs.readFileSync(templateCompilerFullPath, { encoding: 'utf-8' });
-
-    pluginInfo.plugins.forEach(function(plugin) {
-      Compiler.registerPlugin('ast', plugin);
-    });
-
-    var PrecompileInlineHTMLBarsPlugin = HTMLBarsInlinePrecompilePlugin(Compiler.precompile, {
-      cacheKey: [templateCompilerCacheKey].concat(pluginInfo.cacheKeys).join('|')
-    });
+    var cacheKey = [templateCompilerCacheKey].concat(pluginInfo.cacheKeys).join('|');
```

Comment:
Thoughts on sorting this before joining. That way we ensure order of keys is always the same